URI is not absolute

Trying to use this plugin and got this error message when launching the check:

URI is not absolute

Don't know which URI :-/

Sorry didn't notice there was one: 
```
eclipse.buildId=4.6.2.M20161124-1400
java.version=1.8.0_151
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_GB
Framework arguments:  /home/jenkins/workspace/Maxim_Eclipse_ARM/build/maxim-integrated-eclipse-arm-win32-x86_64 /home/jenkins/workspace/Maxim_Eclipse_ARM/build/maxim-integrated-eclipse-arm-win32-x86_64
Command-line arguments:  -os win32 -ws win32 -arch x86_64 /home/jenkins/workspace/Maxim_Eclipse_ARM/build/maxim-integrated-eclipse-arm-win32-x86_64 -data file:/C:/Users/GQ185008/Documents/CODE/workspace/ /home/jenkins/workspace/Maxim_Eclipse_ARM/build/maxim-integrated-eclipse-arm-win32-x86_64

com.googlecode.cppcheclipse.core
Error
Tue Nov 21 15:23:58 GMT 2017
Error checking Keygen

org.eclipse.core.runtime.CoreException: Could not initialize cppcheck
	at com.googlecode.cppcheclipse.ui.Builder$ResourceVisitor.processFile(Builder.java:205)
	at com.googlecode.cppcheclipse.ui.Builder$ResourceVisitor.visit(Builder.java:122)
	at org.eclipse.core.internal.resources.Resource$2.visit(Resource.java:120)
	at org.eclipse.core.internal.resources.Resource$1.visitElement(Resource.java:84)
	at org.eclipse.core.internal.watson.ElementTreeIterator.doIteration(ElementTreeIterator.java:82)
	at org.eclipse.core.internal.watson.ElementTreeIterator.doIteration(ElementTreeIterator.java:87)
	at org.eclipse.core.internal.watson.ElementTreeIterator.doIteration(ElementTreeIterator.java:87)
	at org.eclipse.core.internal.watson.ElementTreeIterator.doIteration(ElementTreeIterator.java:87)
	at org.eclipse.core.internal.watson.ElementTreeIterator.iterate(ElementTreeIterator.java:129)
	at org.eclipse.core.internal.resources.Resource.accept(Resource.java:94)
	at org.eclipse.core.internal.resources.Resource.accept(Resource.java:52)
	at org.eclipse.core.internal.resources.Resource.accept(Resource.java:117)
	at org.eclipse.core.internal.resources.Resource.accept(Resource.java:105)
	at com.googlecode.cppcheclipse.ui.Builder.processResource(Builder.java:306)
	at com.googlecode.cppcheclipse.ui.commands.RunCodeAnalysis$RunCodeAnalysisJob.runResource(RunCodeAnalysis.java:26)
	at com.googlecode.cppcheclipse.ui.commands.ResourceSelectionJob.run(ResourceSelectionJob.java:44)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: org.eclipse.core.runtime.CoreException: Could not initialize cppcheck for project Keygen
	at com.googlecode.cppcheclipse.ui.Builder$ResourceVisitor.initChecker(Builder.java:192)
	at com.googlecode.cppcheclipse.ui.Builder$ResourceVisitor.processFile(Builder.java:199)
	... 16 more
Caused by: java.lang.IllegalArgumentException: URI is not absolute
	at java.io.File.<init>(Unknown Source)
	at com.googlecode.cppcheclipse.ui.ToolchainSettings.resolveIncludePath(ToolchainSettings.java:139)
	at com.googlecode.cppcheclipse.ui.ToolchainSettings.getIncludes(ToolchainSettings.java:195)
	at com.googlecode.cppcheclipse.ui.ToolchainSettings.getUserIncludes(ToolchainSettings.java:87)
	at com.googlecode.cppcheclipse.core.Checker.<init>(Checker.java:80)
	at com.googlecode.cppcheclipse.ui.Builder$ResourceVisitor.initChecker(Builder.java:164)
	... 17 more

```

--- a/com.googlecode.cppcheclipse.ui/src/com/googlecode/cppcheclipse/ui/ToolchainSettings.java
+++ b/com.googlecode.cppcheclipse.ui/src/com/googlecode/cppcheclipse/ui/ToolchainSettings.java
@@ -25,6 +25,7 @@ import com.googlecode.cppcheclipse.core.CppcheclipsePlugin;
 import com.googlecode.cppcheclipse.core.IToolchainSettings;
 import com.googlecode.cppcheclipse.core.Symbol;
 
+
 /**
  * Getting some settings done in the toolchain of CDT
  * 
@@ -136,7 +137,7 @@ public class ToolchainSettings implements IToolchainSettings {
 			// if we could resolve the file
 			if (files.length > 0) {
 				for (IFile file : files) {
-					result.add(new File(file.getRawLocationURI()));
+					result.add(new File(file.getLocationURI()));
 				}
 			} else {
 				// otherwise just take the whole file

